tests/accelerators/test_ddp.py

```diff
@@ -90,11 +91,14 @@ def test_torch_distributed_backend_env_variables(tmpdir):
 
 
 @RunIf(skip_windows=True)
-@mock.patch("torch.cuda.device_count", return_value=1)
-@mock.patch("torch.cuda.is_available", return_value=True)
 @mock.patch("torch.cuda.set_device")
+@mock.patch("torch.cuda.is_available", return_value=True)
+@mock.patch("torch.cuda.device_count", return_value=1)
+@mock.patch("pytorch_lightning.accelerators.gpu.GPUAccelerator.is_available", return_value=True)
 @mock.patch.dict(os.environ, {"PL_TORCH_DISTRIBUTED_BACKEND": "gloo"}, clear=True)
-def test_ddp_torch_dist_is_available_in_setup(mock_set_device, mock_is_available, mock_device_count, tmpdir):
+def test_ddp_torch_dist_is_available_in_setup(
+    mock_set_device, mock_cuda_available, mock_device_count, mock_gpu_is_available, tmpdir
```

Comment:
the order of arguments is incorrect. the decorator closest to the test function corresponds to the first argument, and the one on the very top corresponds to the right-most argument. 

it does not matter here because these are not being used, just as a note.

Reply:
this applies to other test changes below as well.